resources/developer-mode.mjs

```diff
@@ -63,33 +68,56 @@ export function createUIForWarmupSuite() {
     return label;
 }
 
-export function createUIForIterationCount() {
-    let range = document.createElement("input");
-    range.type = "range";
-    range.id = "iteration-count";
-    range.min = 1;
-    range.max = 20;
-    range.value = params.iterationCount;
-
-    let label = document.createElement("label");
-    let countLabel = document.createElement("span");
-    countLabel.textContent = params.iterationCount;
-    label.append("iterations: ", countLabel, document.createElement("br"), range);
-
-    range.oninput = () => {
-        countLabel.textContent = range.value;
+function createUIForIterationCount() {
+    const { range, label } = createTimeRangeUI("Iterations: ", params.iterationCount, "#", 1, 100);
```

Comment:
can we use `200` as a max value ? I think some folks at Mozilla routinely use `200` for their local tests.

Reply:
DONE.